stories/components/NavigationBar.stories.js

```
@@ -0,0 +1,319 @@
+import VsNavigationBarSearch from '@/components/navigation-bar/components/NavigationBarSearch.vue';
+import VsLink from '@/components/link/Link.vue';
+import VsIcon from '@/components/icon/Icon.vue';
+import VsButton from '@/components/button/Button.vue';
+import VsAccordion from '@/components/accordion/Accordion.vue';
+import VsAccordionItem from '@/components/accordion/components/AccordionItem.vue';
+import VsAccordionToggle from '@/components/accordion/components/AccordionToggle.vue';
+import VsDivider from '@/components/divider/Divider.vue';
+import VsTooltip from '@/components/tooltip/Tooltip.vue';
+
+import VsSvgLink from '@/components/svg-link/SvgLink.vue';
+import designTokens from '@/assets/tokens/tokens.json';
```

ESLint

[error] `@/assets/tokens/tokens.json` import should occur after import of `@/assets/fixtures/navigation-bar/many-items-nav.json`

(import/order)